server/graphql/v2/object/Event.js

```diff
@@ -51,6 +52,16 @@ export const Event = new GraphQLObjectType({
           }
         },
       },
+      endsAt: {
+        description: 'The Event end date and time',
+        type: GraphQLDateTime,
+        async resolve(event) {
+          if (!event.endsAt) {
+            return null;
+          }
+          return new Date(event.endsAt);
```

Comment:
What justifies a such complicated resolver? Is a custom resolver actually needed? See `createdAt` on most objects.

Reply:
Yep, I thought we need it to handle the null case, but I am wrong. We don't need it. I've removed the resolver. Thanks for pointing it out. 👍🏽